src/Primary_Extension/org/lobobrowser/primary/ext/TextViewerWindow.java

```diff
@@ -43,6 +49,10 @@
   private final JTextArea textArea;
   private boolean scrollsOnAppends;
 
+  // Key Stroke for CMD in case of Mac and CTRL otherwise
+  private static final int CMD_CTRL_KS = PlatformInit.OS_NAME == OS.MAC ? Toolkit.getDefaultToolkit().getMenuShortcutKeyMask()
```

Comment:
Yeah, but I felt that it belongs to UX and PlatformInit might not be right place to have it there.